docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+services:
+
+  init-private-key:   # create private key
```

Comment:
isn't this creating a private key only to make it accessible to all the nodes? 🤔

Reply:
yes, that was not clear to me: as all I had to go on was a localhost example, all the nodes seem to be using the same private key

https://dedis.github.io/dela/#/memcoin

but I gather that this should not be the case?

Reply:
Ah, I see where you're coming from.
The [BLS signature scheme](https://en.wikipedia.org/wiki/BLS_digital_signature) is used because it allows for signature aggregation. Effectively, each node can use its own signature and the resulting signature is an aggregation of the individual signatures. 

I think out of convenience the demo was using a single key, but separate keys might make sense as a proper example of how the system should work / be used.

Reply:
I'm currently on it, so please don't merge until after I've pushed the necessary changes :slightly_smiling_face: